vio/vio/swagger/nova_utils.py

```diff
@@ -34,8 +34,14 @@ def server_formatter(server, interfaces=[]):
         r['nicArray'] = [{'portId': i.port_id} for i in interfaces]
     elif server.networks:
         r['nicArray'] = [{'portId': n['port']} for n in server.networks]
-    if server.attached_volumes:
-        r["volumeArray"] = [{'volumeId': v['id']} for v in server.attached_volumes]
+    # TODO: wait sdk fix block_device_mapping
```

Comment:
I upload a patch to fix block_device_mapping issue in community https://review.openstack.org/#/c/446931/